### PR TITLE
map22 (skipped 40, 42, 55, 57)

### PR DESCRIPTION
**Pull request by carrotcake227**

### CODE REVIEW

File: Translation Script/Map107.csv
Comment (OfficialLayton) [inline Translation Script/Map107.csv:63]: Ditto for this translation too ^^

File: Translation Script/Map107.csv
Comment (OfficialLayton) [inline Translation Script/Map107.csv:119]: Oh dear, we can't have this. If we remove the commas like this, the csv will break!

That wouldn't be good at all. Definitely go through and fix these if you can haha

File: Translation Script/Map107.csv
Comment (OfficialLayton) [inline Translation Script/Map107.csv:127]: Ditto commas for this.

File: Translation Script/Map107.csv
Comment (OfficialLayton) [inline Translation Script/Map107.csv:136]: Commas.

File: Translation Script/Map107.csv
Comment (OfficialLayton) [inline Translation Script/Map107.csv:144]: Commas.

**Review by OfficialLayton — CHANGES_REQUESTED** [review]
Sorry for taking so long on the review Carrot! Thanks again for your help!
Just a slight translation change request and editing those commas back in would be great.

After those are done I'd be happy to approve. Thanks again ^^

**Comment by carrotcake227** [comment]
Sorry for the delay! School is picking up for me again and I was having trouble with git which kind of demotivated me to work on this haha. I'll try to get through more of it soon. 

Also, when you said commas, did you mean quotation marks? That's what I interpreted it as.

**Review by OfficialLayton — APPROVED** [review]

File: Translation Script/Map107.csv
Comment (OfficialLayton) [inline Translation Script/Map107.csv:30]: More missing quotation marks here too!

File: Translation Script/Map107.csv
Comment (OfficialLayton) [inline Translation Script/Map107.csv:14]: Here as well!

**Review by OfficialLayton — CHANGES_REQUESTED** [review]

**Comment by OfficialLayton** [comment]
Hello @carrotcake227! 

I am very, very sorry for the late reply, I've been busy myself so I can totally understand. Wishing you all the best for your school stuffs! Once again, thank you very, very much for your help. I can't thank you enough! And please, feel free to go at your own pace.  If you're having any more trouble with git in the future let me know and I'd be happy to help. It's the least I can do ^^
Ack, you're totally right. I did mean the quotation marks! Basically without them the csvs will be read incorrectly.

For example: 
`"カチコチに凍っている・・・",It's frozen solid...`
Should be:
`"カチコチに凍っている・・・","It's frozen solid..."`

I did still seem some more cases of missing quotation marks, did you manage to get them all?

**Comment by OfficialLayton** [comment]
Looks good to me!

Sorry about the lateness in getting this merged. Hopefully nothing breaks and thank you very, very much Carrot! Much appreciated ^^

**Comment by OfficialLayton** [comment]
Ah, sorry my bad @carrotcake227 it appears there are some merge conflicts, let me know once they're done and I'll merge after that ^^

**Comment by OfficialLayton** [comment]
Heyo @carrotcake227!

Sorry about the late reply, I know you're busy, so I had a look at the merge conflicts and they *seem* to be just from the comma error that happened recently. I've done my best to resolve them myself so you don't have to worry about it ^^

With this, I should be able to just merge what you've done into the translation! Thanks again for your help :D

**Pull request merged**
